module/modmanager/manager.go

```diff
@@ -94,13 +94,58 @@ type addedResource struct {
 	deps []string
 }
 
+// moduleMap is a typesafe wrapper for a sync.Map holding string keys and *module values.
+type moduleMap struct {
```

Comment:
Nice to have these wrappers to give a bit more type "safety" to the code in `manager.go`!